src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$text-color-first: #060b35;
+$text-color-second: #616070;
+$blue-color: #00acdc;
+$white-color: #fff;
+$light-grey: #f3f3f3;
+$stars-count: 4;
+$body-margin: 0;
```

Comment:
you are using this property once in a project. so it not necessary to keep it, please remove this